ds000117/sub-01/ses-meg/sub-01_ses-meg_scans.tsv

```diff
@@ -1,7 +1,7 @@
 filename	acq_time
-meg/sub-01_ses-meg_task-facerecognition_run-01_meg.fif	##-##-##T12:04:14
-meg/sub-01_ses-meg_task-facerecognition_run-02_meg.fif	##-##-##T12:17:17
-meg/sub-01_ses-meg_task-facerecognition_run-03_meg.fif	##-##-##T12:27:22
-meg/sub-01_ses-meg_task-facerecognition_run-04_meg.fif	##-##-##T12:37:33
-meg/sub-01_ses-meg_task-facerecognition_run-05_meg.fif	##-##-##T12:47:23
-meg/sub-01_ses-meg_task-facerecognition_run-06_meg.fif	##-##-##T12:58:06
+meg/sub-01_ses-meg_task-facerecognition_run-01_meg.fif	2018-05-05T12:04:14
```

Comment:
This is incorrect, no? Is this the date you get from the fif file?

Reply:
@jasmainak the fif files corresponding to these _scans.tsv files are empty. How would I get a date from that file?

https://github.com/INCF/BIDS-examples/blob/master/ds000117/sub-01/ses-meg/meg/sub-01_ses-meg_task-facerecognition_run-01_meg.fif.gz

Reply:
here is the full data: https://openneuro.org/datasets/ds000117. You could use MNE to get the date. But I guess it doesn't matter if the dates are accurate or not ... this repo is only for testing the validator

Reply:
But maybe use a very old date so it is clear that the dates are dummy?

Reply:
sure thing